test-framework/arquillian/src/main/java/io/quarkus/arquillian/MethodParameterInjectionPoint.java

```diff
@@ -0,0 +1,114 @@
+package io.quarkus.arquillian;
+
+import java.lang.annotation.Annotation;
+import java.lang.reflect.Member;
+import java.lang.reflect.Method;
+import java.lang.reflect.Type;
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+
+import javax.enterprise.inject.Default;
+import javax.enterprise.inject.spi.Annotated;
+import javax.enterprise.inject.spi.AnnotatedCallable;
+import javax.enterprise.inject.spi.AnnotatedParameter;
+import javax.enterprise.inject.spi.Bean;
+import javax.enterprise.inject.spi.BeanManager;
+import javax.enterprise.inject.spi.InjectionPoint;
+import javax.enterprise.util.AnnotationLiteral;
+
+public class MethodParameterInjectionPoint<T> implements InjectionPoint {
+    private Method method;
+    private int position;
+    private BeanManager beanManager;
+
+    public MethodParameterInjectionPoint(Method method, int position, BeanManager beanManager) {
+        this.method = method;
+        this.position = position;
+        this.beanManager = beanManager;
+    }
+
+    public Bean<?> getBean() {
+        return null;
+    }
+
+    public Member getMember() {
+        return method;
+    }
+
+    public Set<Annotation> getQualifiers() {
+        Set<Annotation> qualifiers = new HashSet<>();
+        // TODO - how to properly determine which annotations are qualifiers? beanManager.isQualifier() can't be used
```

Comment:
Filtering annotations annotated with `@Qualifier` should work in 90% use cases...